### PR TITLE
Content: Changes for Home Page

### PR DESCRIPTION
**Pull request by shredtechular**
This PR includes content changes (mostly...) that were requested in #32.

I wasn't completely clear what a few items meant, but did my best to do them as I thought best.  Below are line items of each requested item and a screen cap of before/after the edits.

# Line item changes requested in #32 

I assume this meant the page title, or really even the part of the title that appears on EVERY page?  
So, that change was made in the `_config.yml` file which is a good place to keep something like that.
### Before
screen shot 2017-10-08 at 5 07 24 pm

### After
screen shot 2017-10-08 at 5 12 22 pm

Unlocking the benefits of authentic connection experiences to empower sustainable healing after detox from heroin & opioid addiction.
**to:**
Detox aftercare services that liberate human beings from opioid addiction relapse.

### Before
screen shot 2017-10-08 at 5 14 47 pm

### After
screen shot 2017-10-08 at 5 14 38 pm

I have been a seeker and still am
but I stopped asking the books and the stars.
I started listening to the teaching of my soul.

### After
screen shot 2017-10-08 at 5 16 22 pm

SeekHealing is a detox aftercare model that unlocks the neurochemical benefits of authentic connection experiences. This non-profit organization provides equal-opportunity programs that use principles of connection & community to facilitate sustainable healing after detox from opioid & heroin addiction.
**to this, in a slightly larger font:**
A detox aftercare model that unlocks the neurochemical benefits of authentic connection experiences. This non-profit organization provides equal-opportunity programs that use principles of connection & community to empower sustainable healing after detox from opioid & heroin addiction.
**then please add another little divider line and then this, in a slightly smaller font:**
Founded in 2016, SeekHealing is a 501(c)3 charity and post-detox recovery model that challenges mainstream stigmas about opioid and heroin addiction. Our programs turn the idea of standard rehab upside down by providing affordable services that prioritize authentic connection and harm reduction. <....more>

### Before
screen shot 2017-10-08 at 5 19 51 pm

### After
screen shot 2017-10-08 at 5 19 44 pm

### CODE REVIEW

**Comment by howtohumanbetter** [comment]
PERFECT! This is exactly what was needed. Thank you, Andrew.

**Comment by shredtechular** [comment]
@j00bar this PR is ready to be reviewed now.

File: index.html [hunk 1]
```diff
@@ -165,7 +177,17 @@ <h1 class="main-header hide-under-1024" data-caption-animate="fadeInUp">It's tim
...
+						</div>
+					</div>
+					<div class="heading-block center"></div>
+					<div class="row">
+						<div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2 col-xs-12 text-center">
+							<p>Founded in 2016, SeekHealing is a 501(c)3 charity and post-detox recovery model that challenges mainstream stigmas about opioid and heroin addiction. Our programs turn the idea of standard rehab upside down by providing affordable services that prioritize authentic connection and harm reduction.</p>
+
+							<p>Our programs are designed for the 6-12 months after detox that are most critical to sustainable recovery and mental health integration. The methodology is focused to enhance the neuroplasticity of participants through: intentional experiences of authentic social & spiritual connection; rewarding the practice of learning new things; and facilitating a major change to their daily environment.</p>
```
Comment (j00bar) [inline index.html]: Sorry - same HTML entity comment. 🙄 Thanks for accommodating my pedantry. There's one two lines later too.

Comment (shredtechular) [reply index.html]: No worries at all, I converted all the Ampersands now in b2321aa

**Comment by j00bar** [comment]
Tried resolving conflicts internally to no avail. Whichever of us gets to this first will fix and we'll merge.

**Comment by shredtechular** [comment]
@j00bar I rebased master and resolved the conflicts.  This is ready for your review again.

**Pull request merged**